exercises/proverb/.meta/gen.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"log"
+	"text/template"
+
+	"../../../gen"
+)
+
+func main() {
+	t, err := template.New("").Parse(tmpl)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var j js
+	if err := gen.Gen("proverb", &j, t); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// The JSON structure we expect to be able to unmarshal into
+type js struct {
+	Cases []struct {
+		Description string
+		Cases       []struct {
```

Comment:
I see the generator isn't working correctly yet to make cases_test.go.
First problem is the canonical-data.json only has 1 level of "cases". Can drop the inner Cases here.